cypress.config.js

```diff
@@ -1,4 +1,4 @@
-const { defineConfig } = require('cypress');
+const { defineConfig } = require('cypress')
```

Comment:
In this case, take a look at your Prettier extension, it seems something is off there. This one shouldn't be modified if the extension is reading the Prettier project config.